docs/FAQ.rst

```diff
@@ -722,6 +739,9 @@ Work order response
     A response generated by an Avalon worker to a work
     order. The response is a JSON string
 
+WPE
+    Work order Processor Enclave Avalon worker. Compare with KME and WPE
```

Comment:
It should be "Compare with KME and _**Singleton**_"
I would add (before the phrase above) "WPE is a part of the worker pool responsible for the workorder processing. It doesn't has access to the worker's private keys and relies on the KME to provide necessary context for the workorder processing."